docs/core/testing/index.md

```diff
@@ -1,23 +1,27 @@
 ---
 title: Unit Testing in .NET Core
-description: Unit testing has never been easier. See how to use unit testing in .NET Core projects.
-keywords: .NET, .NET Core
+description: Unit testing has never been easier. See how to use unit testing in .NET Core and .NET Standard projects.
+keywords: .NET, .NET Core, .NET Standard, unit testing
 author: ardalis
 ms.author: wiwagn
-ms.date: 06/20/2016
+ms.date: 08/30/2017
 ms.topic: article
 ms.prod: .net-core
 ms.devlang: dotnet
 ms.assetid: 815ac74c-4bd9-4a94-a87c-78288b27c0e2
 ---
 
-# Unit Testing in .NET Core
+# Unit Testing in .NET Core and .NET Standard
 
-.NET Core has been designed with testability in mind, so that creating unit tests for your applications is easier than ever before. This article briefly introduces unit tests (and how they differ from other kinds of tests). Linked resources demonstrates how to add a test project to your solution and then run unit tests using either the command line or Visual Studio.
+.NET Core has been designed with testability in mind, so that creating unit tests for your applications is easier than ever before. This article briefly introduces unit tests (and how they differ from other kinds of tests). Linked resources demonstrate how to add a test project to your solution and then run unit tests using either the command line or Visual Studio.
+
+.NET Core 2.0 supports [.NET Standard 2.0](../../standard/net-standard.md). The libraries used to demonstrate unit testing in this section rely on .NET Standard and will work in other project types as well.
+
+Beginning with .NET Core 2.0, there are unit test project templates for VB.NET and F# as well as C#.
```

Comment:
one little nit thing: I think throughout the docs we usually refer to VB as Visual Basic. Should we be consistent on that?